tests/conftest.py

```diff
@@ -304,6 +297,7 @@
     os.path.join(CUR_PATH, "data", "exception_skill", SKILL_YAML),
 ]
 
```

Comment:
You seem to have accidentally deleted some code here, please revert

Reply:
Weird. Done.